pipeline-modules/deployment-service/azure/azure-functions.yaml

```diff
@@ -46,8 +46,20 @@ inputs:
       title: Job Type
       description: This is to deploy or undeploy application
       type: string
-      default: deploy
-  required: ["function_app_name"]
+      default: deploy       
+    FeedName:
+      title: Artifact Feed Name
+      description: Azure devops Artifact Feed name where App artifact will download 
+      type: string    
+    ServiceConnectionString:
```

Comment:
I think better to change it to something like sc_string or service_connection_string